apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -122,4 +122,248 @@ private Operation PollForCompletion(Operation operation, string alias)
             return operation;
         }
     }
+    [Collection(nameof(ComputeFixture))]
+    public sealed class InstancesTests{
```

Comment:
Please use brace-on-new-line formatting - see the existing code for examples.

Reply:
(If you're using an IDE, I'd expect that just autoformatting the whole document should be fine.)

Reply:
done. autoformatted via Resharper CLI tools.